### DISCUSSION TITLE
Oqtane 4.0.6

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#3381] [General]
As per the information shared at the previous Developer Meeting, Oqtane 4.0.6 will be released this week. It will be a stabilization release and will be the last 4.x release on .NET 7 (STS).

The next release will be Oqtane 5.0 and will be based on .NET 8 (LTS). The goal is to release Oqtane 5.0 on ~Nov 15 (immediately after .NET 8 officially ships). Oqtane 5.0 will be focused on .NET 8 compatibility and regression testing of all current scenarios. It may also embrace some of the new Blazor rendering modes in .NET 8, however this will be dependent on the outcome of research to be conducted over the coming month.

**Comment by sbwalker** [comment]
Release announcement : https://www.oqtane.org/blog/!/74/oqtane-4-0-6-released

**Reply by leigh-pointer** [reply]
@sbwalker Update is not showing in the System Update?
![updateIssue](https://github.com/oqtane/oqtane.framework/assets/9447612/244cd4d9-762f-4f58-8e51-c96bdf90d0ab)

**Reply by sbwalker** [reply]
@leigh-pointer it is working for me:

![image](https://github.com/oqtane/oqtane.framework/assets/4840590/784c50f1-b9a5-4cb9-836c-8a51831219da)

Please note that a change was made in 4.0.4 to prevent System Updates in a localhost environment:

https://github.com/oqtane/oqtane.framework/pull/3268

This was done because developers have attempted to use the System Update feature to upgrade their local development environment - which is not a supported scenario ( the System Update only replaces DLLs and static assets - not code assets).

**Reply by leigh-pointer** [reply]
@sbwalker ok, this is a compiled site running in IIS.  I will just upload the package.

**Reply by sbwalker** [reply]
@leigh-pointer what is the Url you use to access the IIS site?

**Reply by leigh-pointer** [reply]
@sbwalker   Module.local and oqtane.local

**Reply by sbwalker** [reply]
@leigh-pointer ok the issue is not related to #3268 then as the code specifically looks for a Url containing "localhost:". Is it possible that you changed the PackageRegistryUrl in appsettings.json to point at the Sandbox? If so, this would explain the behavior as the Sandbox is a completely different registry and was not updated as part of the 4.0.6 release.

**Reply by leigh-pointer** [reply]
@sbwalker yes I did, module.local is looking at sandbox.  Thanks!
